css/index.css

```diff
@@ -71,12 +71,61 @@ p {
     margin: 0 auto;
 }
 
+
+
+
+
+
+
+
+
+#header1{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
```

Comment:
Instead of space-evenly you could use space-between on your header1 selector and it will push the logo and nav apart all the way to the edges of the header, this would look a little closer to the design file.